test/forkMainnet/LimitOrderSwap/GroupFill.t.sol

```diff
@@ -233,7 +215,8 @@ contract GroupFillTest is LimitOrderSwapTest {
         // scenario
         // maker tokens (input): 5 WETH, 8000 USDT
         // taker tokens (output): 1 ETH, 3 WETH, 8000 USDT
-        // profit for arbitrageur: 1 ETH/WETH
+        // should unwrap 4 ether and leave 1 WETH
```

Comment:
I think this case unwraps only 2 WETH (5 WETH - 3 WETH), and 1 ETH for order and another 1 ETH for arbitrageur's profit, right?

Reply:
Yeah, you're right. Unwrap 3 and profit is 1.

Reply:
Fixed in 0708ce1